src/ContinuousEnergyNeutron.tex

```diff
@@ -154,27 +157,34 @@ \subsubsection{\textsf{NU} Block}\label{sec:NUBlock}
+For the \Block{DNU}, the delayed \nubar\ array begins at \xss{KNU} where {\sffamily \var{KNU} = \jxs{24}}. Delayed \nubar\ must be given in the tabulated form as described in \Tableref{tab:NUBlockTabulated}. The polynomial form is not allowed in the \Block{DNU}.
+
+\subsubsection{\textsf{BDD} Block}\label{sec:BDDBlock}
+
+The \Block{BDD} is used to specify basic delayed neutron precursor data and is present only if \jxs{25} > 0. For every precursor group (the total number of precursor groups is given in \nxs{8}), a decay constant is given along with the partial probability that a delayed fission neutron is born from the current group. This data is given in the format given in table \Tableref{tab:DelayedPrecursorDistribution}.
+
 \begin{ThreePartTable}
   \begin{TableNotes}
     \item[$\dagger$] \label{tn:schemeDelayedPrecursors} If $N_{R}=0$, \var{NBT} and \var{INT} are omitted and linear-linear interpolation is assumed.
   \end{TableNotes}
   \begin{XSSTable}{Delayed \nubar\ precursor distribution.}
-    \startblock{DNU}                   & \var{DEC}$_{i}$                  & Decay constant for the $i$-th group \\
-    \startblock{DNU}+1                 & $N_{R}$                          & Number of interpolation regions \\
-    \startblock{KNU}+2                 & \var{NBT}$(l), l=1,\ldots,N_{R}$ & ENDF interpolation parameters\tnotex{tn:schemeDelayedPrecursors} \\
-    \startblock{KNU}+2+$N_{R}$         & \var{INT}$(l), l=1,\ldots,N_{R}$ & ENDF interpolation scheme \\
-    \startblock{DNU}+2+$2N_{R}$       & $N_{E}$                          & Number of energies \\
-    \startblock{DNU}+3+$2N_{R}$       & $E(l),l=1,\ldots,N_{E}$          & Tabulated energy points \\
-    \startblock{DNU}+3+$2N_{R}+N_{E}$ & $P(l),l=1,\ldots,N_{E}$          & Corresponding probabilities
+    \var{KNU}                   & \var{DEC}$_{i}$                  & Decay constant for the $i$-th group \\
+    \var{KNU}+1                 & $N_{R}$                          & Number of interpolation regions \\
+    \var{KNU}+2                 & \var{NBT}$(l), l=1,\ldots,N_{R}$ & ENDF interpolation parameters\tnotex{tn:schemeDelayedPrecursors} \\
+    \var{KNU}+2+$N_{R}$         & \var{INT}$(l), l=1,\ldots,N_{R}$ & ENDF interpolation scheme \\
+    \var{KNU}+2+$2N_{R}$        & $N_{E}$                          & Number of energies \\
+    \var{KNU}+3+$2N_{R}$        & $E(l),l=1,\ldots,N_{E}$          & Tabulated energy points \\
+    \var{KNU}+3+$2N_{R}+N_{E}$  & $P(l),l=1,\ldots,N_{E}$          & Corresponding probabilities
     \label{tab:DelayedPrecursorDistribution}
   \end{XSSTable}
   \begin{tablenotes}
-    \note \startblock{DNU} is the index of the \XSS\ array where the delayed \nubar\ precursor distribution begins; the first one is at \startblock{DNU}=\jxs{25}.
+    \note \var{KNU} is the index of the \XSS\ array where the delayed \nubar\ precursor data for each group begins; the first one is at \var{DNU} = \jxs{25}.
```

Comment:
What the table (and the following note) don't really make clear is where the data for the second precursor group is and beyond. One can take an educated guess that they follow the first group, but it doesn't say that explicitly. I think maybe this would be clearer if the table above were closer in format to what we have for, e.g., Table 30 and Table 38, where the same format is used for multiple energies (in this case multiple groups) and the table makes that explicit. Thoughts?

Reply:
That's an idea. I'll try to make it happen.

Reply:
@paulromano I updated the document with a solution for this.